app/src/components/fineTunes/ContentTabs/ContentTabs.tsx

```diff
@@ -1,8 +1,10 @@
 import { useState, useRef, useEffect, forwardRef } from "react";
-
 import { Button, VStack, HStack, Text, Divider, Box } from "@chakra-ui/react";
+
 import General from "./General/General";
 import TrainingData from "./TrainingData/TrainingData";
+import TestingData from "./TestingData/TestingData";
```

Comment:
Should we rename everything from `TestingData` to `TestSet` to be consistent with what other people call it?

Reply:
I don't feel strongly. I'll rename it.